style.css

```diff
@@ -102,3 +113,296 @@ button:active {
   background: #2230d2;
   color: #fff;
 }
+
+/* Works */
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 88px;
+  position: absolute;
+  width: 375px;
+  height: 2788px;
+  top: 718px;
+  background: #dfe1e6;
+}
+
+.card-works1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-works2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 596px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```

Comment:
- [ ] Please kindly avoid code repetition as pointed earlier. We need to follow DRY CSS principle, the main idea with DRY in CSS is to avoid repeating the same bits of code in our style sheet. Kindly follow the suggestion given by the previous reviewer. ℹ️

Reply:
@OyePriscilla @brainconnect93, I will improve on this rule subsequently. Using the suggested css changes a lot of things in the design.